bin/html2bemjson.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+
+var pkg = JSON.parse(fs.readFileSync(__dirname + '/../package.json'));
+var lib = require('../');
+
+require('coa').Cmd()
+    .name(process.argv[1])
+    .title('html to bemjson converter')
+    .helpful()
+
+    .opt()
+        .name('version')
+        .title('Version')
+        .short('v')
+        .long('version')
+        .flag()
+        .act(function(opts) {
+            return pkg.version;
+        })
+        .end()
+
+    .opt()
+        .name('preserveComments')
+        .title('Preserve comments')
+        .long('with-comments')
+        .def(false)
+        .end()
+
+    .opt()
+        .name('preserveWrongClasses')
+        .title('Preserve unparsable classes')
+        .long('with-wrong-classes')
```

Comment:
@tadatuta I'm thinking about `--without-wrong-classes` to be consistent with library defaults. And also there are no naming options ;-( atm.